external/storm-kafka-client/src/main/java/org/apache/storm/kafka/trident/TridentKafkaState.java

```diff
@@ -116,8 +114,9 @@ public void updateState(List<TridentTuple> tuples, TridentCollector collector) {
             }
 
             if (exceptions.size() > 0) {
-                StringBuilder errorMsg = new StringBuilder("Could not retrieve result for messages " + tuples + " from topic = " + topic
-                    + " because of the following exceptions:" + System.lineSeparator());
+                StringBuilder errorMsg = new StringBuilder("Could not retrieve result for messages ");
```

Comment:
Since the message is only ~5 strings long, I don't think there's much reason to use a StringBuilder at all.

Reply:
There's a foreach statement right below it to iterate the exceptions, and append each exception to the errorMsg, it is still necessary I think.

for (ExecutionException exception : exceptions) {
    errorMsg = errorMsg.append(exception.getMessage()).append(System.lineSeparator());
}

Reply:
Makes sense, thanks.